runner/runner.go

```diff
@@ -110,7 +124,7 @@ func main() {
 	gEventRetrieversRunning = make([]bool, numGetWorkers)
 	gEventWorkersRunning = make([]bool, numRunWorkers)
 
-	if gMetricsListenAddr != "" {
+	if !fpm && gMetricsListenAddr != "" {
```

Comment:
why do we disable metrics when using fpm?

Reply:
My impression was that if fpm is being used we wouldn't be getting any useful metrics anyways. If we want to allow it to listen and grab whatever metrics are getting updated I'm ok with that.